pkgs/development/python-modules/hikari/README.md

```diff
@@ -0,0 +1,5 @@
+# python3Packages.hikari
+## Updating
+The package build process requires a Git commit hash. This has been semi-automated by a script.
```

Comment:
If the suggestion from the update.sh are implemented at least on my opinion we could remove the readme, as we are not giving any needed information here.

Updates for the package should be done by `nix-shell maintainers/scripts/update.nix --argstr package python3Packages.hikari` and with my suggestions that update.sh does all the handling

Reply:
Unused, see https://github.com/NixOS/nixpkgs/pull/232557#discussion_r1542525480